desafio-02/roberto0arruda/python/README.md

```diff
@@ -0,0 +1,22 @@
+# 2: PRIMOS
+
+## Listando números primos
+
+### Desafio
+
+Escrever um programa para listar todos os números primos entre 1 e 10000, na linguagem de sua preferência.
+
+### Solução
+
+``` python
+for num in range(1, 10000 + 1):
+    tot = 0
+    for n in range(1, num + 1):
+        if num % n == 0:
+            tot += 1
+
+    if tot == 2:
```

Comment:
Aqui nem precisaria desse teste, porque voce só chega aqui se o loop anterior passar por todos sem dar break. Olhe o "else" no for loop do python.

Reply:
@marcopaganini muito obrigado pelas dicas, li a respeito do "else" no for loop do python e já implementei as devidas correções.